ui/shared/utils/constants.js

```diff
@@ -550,6 +550,42 @@ const ORDERED_VEP_CONSEQUENCES = [
     value: 'DUP',
     group: VEP_GROUP_SV,
   },
+  {
+    description: 'A translocation variant',
+    text: 'Translocation',
+    value: 'BND',
```

Comment:
`BND` seems like a weird abbreviation for a translocation, so just double checking that this isn't a typo

Reply:
It is from the VCF. I checked online and couldn't find why `BND` stands for a translocation. But it is not a type.